### PR TITLE
Get leaderboard v0.1

### PR DESCRIPTION
**Pull request by GiovanniGobbs**
- Initial version of Get leaderboard
- Refactoring

 **89% coverage**

### CODE REVIEW

File: utb/views/get_article.py [hunk 1]
```diff
@@ -1,46 +1,17 @@
...
-    user = User(id=user_id,
-                name=name,
-                surname=surname,
-                email=email)
-    user.save()
-    return HttpResponse("User added", status=201)
-
-
-def get_article(request):
+def handler(request):
```
Comment (matteofiordarancio) [inline utb/views/get_article.py:7]: why did you change get_article in handler? Doesn't the latter sounds too much as a generic name?

File: utb/views/get_leaderboard.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+from django.http import HttpResponse, JsonResponse
+
+from utb import utils
+from utb.models import User
+from utb.utils import user_exists
+
+
+def handler(request):
```
Comment (matteofiordarancio) [inline utb/views/get_leaderboard.py:8]: Why is everything called handler?

File: utb/views/submit_report.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    article = qs[0]
+
+    report = Report(user=user,
+                    article=article,
+                    report_value=report)
+    report.save()
+    if report == "L":
+        article.positive_reports += 1 * user.multiplier
+    else:
+        article.positive_reports += 1 * user.multiplier
```
Comment (matteofiordarancio) [inline utb/views/submit_report.py]: This two lines are the same

**Review by matteofiordarancio — CHANGES_REQUESTED** [review]

File: utb/views/get_article.py (hunk 1 above)
Comment (GiovanniGobbs) [reply utb/views/get_article.py:7]: Because I edited the views file configuration in order to avoid having every function in the same file

**Review by matteofiordarancio — APPROVED** [review]
